serving/installing-logging-metrics-traces.md

```diff
@@ -153,6 +151,35 @@ To configure and setup monitoring:
      ```
 
   CTRL+C to exit watch.
+
+## Stackdriver for both logging and metrics in GKE
+
+You can enable Stackdriver in the GCP project by following [this](https://cloud.google.com/monitoring/workspaces/guide#single-project-ws).
+When Knative is deployed to GKE, you can choose to send both logging and metrics to Stackdriver backend. To have a complete monitoring experience, follow these steps:
+
+1. To have [Kubernetes metrics](https://cloud.google.com/monitoring/api/metrics_other#other-kubernetes.io), create a Kubernetes cluster on GKE with [Stackdriver Kubernetes Monitoring](https://cloud.google.com/kubernetes-monitoring/) feature:
```

Comment:
I don't think we should be talking about gcloud / GKE here. How would I enable this if I was running Knative somewhere else? Would you still be able to use Stackdriver?

Reply:
Maybe it's unavoidable?

Reply:
Maybe I should describe stackdriver metrics in general, but specialize this section for GKE only.